spec/glossary.adoc

```diff
@@ -11,7 +11,9 @@
 [[connection]]Connection:: A channel of communication between SonarLint and a particular SonarQube server or SonarCloud. See also <<connected_mode,connected mode>>.
 [[connection_config]]Connection Configuration:: All attributes that will be used to instantiate and operate a <<connection,connection>>. Usually provided by the IDE settings. It includes credentials, endpoint URL, timeouts, proxy...
 [[effective_rule_details]]Effective Rule Details:: <<rule_definition>> but taking into account the <<binding,binding>> and the <<finding_context,finding context>>. Some attributes can be different from the <<rule_definition, rule definition>>.
-[[finding]]Finding:: Generic concept that includes all things reported by <<sonar_analyzer,Sonar analyzers>>: issues, security hotspots, taint vulnerabilities, ...
+[[finding]]Finding:: Generic concept that includes all things reported by <<sonar_analyzer,Sonar analyzers>>: issues, security hotspots, taint vulnerabilities, ... With the new Clean Code Taxonomy, security hotspots should disappear, and so everything will be an issue, so this concept might disappear.
+[[issue_matching]]Issue Matching:: Refer to the process, algorithm or xref:issue_tracking.adoc#matching_heuristic[heuristic] that we use to decide if one or multiple issues reported by an analysis are the same as the ones previously reported. Important part of <<issue_tracking,issue tracking>>.
```

Comment:
> decide if one or multiple issues reported by an analysis are the same as the ones previously reported

The wording gives the impression that the output of the process is just a yes or no, but it's more than that. What about:

> the process that takes freshly detected issues and tries to match them with already known issues. The result is a list of matched issues and a list of non-matched issues (they are for example new).